libmscore/input.h

```diff
@@ -79,8 +88,11 @@ class InputState {
       int string() const                  { return _string;             }
       void setString(int val)             { _string = val;              }
 
-      bool repitchMode() const            { return _repitchMode;    }
-      void setRepitchMode(bool val)       { _repitchMode = val;     }
+      bool repitchMode() const            { return _noteEntryMethod == NoteEntryMethod::REPITCH;    }
+      void setRepitchMode(bool val)       { if(val)
+                                                setNoteEntryMethod(NoteEntryMethod::REPITCH);
+                                            else
+                                                setNoteEntryMethod(NoteEntryMethod::STEPTIME);}
```

Comment:
Could be `{ setNoteEntryMethod(val ? NoteEntryMethod::REPITCH : NoteEntryMethod::STEPTIME); }`